lib/lytics/index.js

```diff
@@ -62,7 +63,10 @@ Lytics.prototype.loaded = function(){
  */
 
 Lytics.prototype.page = function(page){
-  window.jstag.send(page.properties());
+  var props = page.properties();
+  props._e = props.name;
+  delete props.name;
```

Comment:
If I can't rely on `.name` how should I go about the remapping of `name` to `_e`?

As for not using `delete`, that's fine. I assumed `page.properties()` returned a copy, but if it doesn't, and what I am doing is destructive, I can change it. 

Is the fold-left you are using coming from somewhere within `analytics.js`, or are you using it purely for example?